assets/src/edit-story/components/dropDownMenu/index.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { useCallback, useRef, useState } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { More } from '../button';
+import DropDownList from '../form/dropDown/list';
+import Popup from '../popup';
+import { useKeyDownEffect } from '../keyboard';
+
+const MoreButton = styled(More)`
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  width: 28px;
+  height: 28px;
+  padding: 0;
+  background: ${({ theme }) => theme.colors.bg.panel};
+  color: ${({ theme }) => theme.colors.fg.white};
+  border-radius: 100%;
+  border: 0;
+`;
+
+const DropDownContainer = styled.div`
+  margin-top: 10px;
+`;
+
+function DropDownMenu({ options, onOption }) {
```

Comment:
Curious, what are the differences for not using the same dropdown menu for Media Library dropdown menu and here?

Reply:
It **is** a direct copy. The media library dropdown menu is however specific to the media item use-case. I created a generalized version from that, and am considering backporting this generalized component to the media library element.

There are slight differences in the interaction model, that I'll have to figure out how to consolidate. The primary being that in the media library the menu is opened when the media item as a whole is focused and you press <kbd>space</kbd> (whereas <kbd>enter</kbd> inserts the item). Also, you cannot actually focus the menu button in the media library - only the media item as a whole.

For the poster image, I have added the menu to the tab order as you'd "normally" do and it can be focused. When focused pressing either <kbd>space</kbd> or <kbd>enter</kbd> will open it (as will <kbd>down</kbd> actually).

I am open to suggestions on how to merge the two interaction models.

Reply:
This consideration is also listed as a TODO by the way 😉 

![Screen Shot 2020-08-20 at 10 03 53](https://user-images.githubusercontent.com/637548/90781242-9210c600-e2cc-11ea-80ae-6ff457ab7437.png)

Reply:
After much consideration, I will not be doing this work here. Let's wait until Accessibility Review is in before we change how this works. There's no pretty way to make the same component work in the two very different ways without creating a very bloated central component.